build/lint.sh

```diff
@@ -2,4 +2,4 @@
 
 set -euox pipefail
 
-build/tester.sh ct lint --config build/ct.yaml --all
+build/tester.sh ct lint --config build/ct.yaml --all --validate-maintainers=false
```

Comment:
Without this, was hitting the below error:
```
Validating maintainers...

------------------------------------------------------------------------------------------------------------------------
 ✖︎ cockroachdb => (version: "14.0.5", path: "cockroachdb") > could not parse git repository domain for "origin"
------------------------------------------------------------------------------------------------------------------------
Error: failed linting charts: failed processing charts
failed linting charts: failed processing charts
make: *** [test/lint] Error 1
```

The issue https://github.com/helm/chart-testing/issues/464 captures the discussion around the resolution of this error. I did not find validating maintainers as a necessary thing, hence choosing to skip the same. Open to thoughts.

Reply:
I agree skipping this check is better than adding some wrapper on chart-testing.